### ISSUE TITLE
Custom Partner Links

### ISSUE THREAD

**Issue by SmokE-PGF** [cswendrowski/HumbleBundleBot#4]
When registering a webhook with custom partner links it only appends our partner links after yours breaking the refferal to humble bundle from working.

On discord the links come up as https://www.humblebundle.com/monthly?partner=cswendrowski?partner=OURPARTNERCODE

**Comment by cswendrowski** [comment]
@SmokE-PGF thanks for reporting this issue! I'll take a look at it as soon as I can

**Comment by cswendrowski** [comment]
@SmokE-PGF I just pushed up a fix for this issue, the next bundles should be resolved.
The issue was as follows:

When the service was first written all messages were always the same, so the message was constructed once per platform (Discord and JSON) then sent to all Webhooks.
When the Partner feature was added, I naively applied it inside of the loop that sends webhooks, which adjusted the URL for all webhooks after it in the Queue. Webhook 1 with Partner 1 would get `?partner=1`, Webhook 2 with no Partner would get `?partner=1`, Webhook 3 with Partner 3 would get `?partner=1?partner=3`, and so on

I have resolved this issue by recreating the message for each Webhook - this will allow for potential future customization of the messages if the service ends up going that way.

Thanks again for reporting this, and I hope you're enjoying the bot!
